TypeScript.ContractGenerator/TypeScriptUnit.cs

```diff
@@ -39,12 +40,13 @@ public TypeScriptVariableReference AddSymbolImport(string symbolName, string pat
                         TypeName = symbolName,
                         CurrentUnit = this,
                         PathToUnit = path,
+                        UseTypeKeyword = false,
                     });
             }
             return new TypeScriptVariableReference(symbolName);
         }
 
-        public TypeScriptVariableReference AddDefaultSymbolImport(string localName, string path)
+        public TypeScriptVariableReference AddDefaultSymbolImport(string localName, string path, bool useTypeKeyword=false)
```

Comment:
AddDefaultSymbolImport and TypeScriptImportDefaultFromPathStatement is never used and can be removed